python/podio_schema_evolution.py

```diff
@@ -327,6 +425,33 @@ def heuristics(self):
         added_members = [change for change in schema_changes if isinstance(change, AddedMember)]
         self.heuristics_members(added_members, dropped_members, schema_changes)
 
+        for vmc in (c for c in schema_changes if isinstance(c, AddedVectorMember)):
+            self.errors.append(
+                f"Forbidden schema change in '{vmc.klassname}': Added vector member '{vmc.member}'"
+            )
+        for vmc in (c for c in schema_changes if isinstance(c, DroppedVectorMember)):
+            self.errors.append(
+                f"Forbidden schema change in '{vmc.klassname}': Added vector member '{vmc.member}'"
+            )
+
+        for rc in (c for c in schema_changes if isinstance(c, AddedSingleRelation)):
+            self.errors.append(
+                f"Forbidden schema chage in '{rc.klassname}': Added OneToOneRelation '{rc.member}'"
```

Comment:
```suggestion
                f"Forbidden schema change in '{rc.klassname}': Added OneToOneRelation '{rc.member}'"
```
Missing `n`s added in my other comment